patches/chrome-common-channel_info_win.cc.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/chrome/common/channel_info_win.cc b/chrome/common/channel_info_win.cc
-index 8a9e2d3335b6cf945864a88d1644e1935234990b..cae0e9f8f7e1d9dc3d2489111cbb8f5d75330928 100644
+index 8a9e2d3335b6cf945864a88d1644e1935234990b..f442eabe6ecad373f11eb84eb91fcd2ae5dfc538 100644
 --- a/chrome/common/channel_info_win.cc
 +++ b/chrome/common/channel_info_win.cc
 @@ -13,7 +13,7 @@
  namespace chrome {
  
  std::string GetChannelName() {
```

Comment:
candidate for chromium_src?

Reply:
Indeed, will work on this (and other cases) in follow-up PRs if that's ok

Reply:
Tracked in https://github.com/brave/brave-core/pull/7996